### PR TITLE
Add some new window api for desktop

### PR DESCRIPTION
**Pull request by mrxiaozhuox**
- Print: open print modal
- SetZoomLevel: change webview zoom level

### CODE REVIEW

File: examples/window_zoom.rs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+fn main() {
+    dioxus::desktop::launch(app);
+}
+
+fn app(cx: Scope) -> Element {
+    let window = use_window(&cx);
+
+    let level = use_state(&cx, || 1.0);
+
+    window.set_zoom_level(*level.get());
```
Comment (rMazeiks) [inline examples/window_zoom.rs:13]: looks like this sets the zoom level on every re-render?

in this example its kinda fine, but people might copy-paste it into their own work and that won't be great

may i suggest calling this inside the event handling closure?

File: examples/window_zoom.rs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    cx.render(rsx! {
+        input {
+            r#type: "number",
+            value: "{level}",
+            oninput: |e| {
+                level.set(e.value.parse::<f64>().unwrap_or_default())
```
Comment (rMazeiks) [inline examples/window_zoom.rs:19]: default for f64 is 0; is this what you want?

you could also `unwrap_or(1.)`

**Review by rMazeiks — COMMENTED** [review]
looks quite useful!

**Review by jkelleyrtp — APPROVED** [review]
Looks good - thanks for adding this! :)

**Pull request merged**